helm/alfresco-identity-service/values.yaml

```diff
@@ -74,7 +77,7 @@ keycloak:
       port: 80
     extraInitContainers: |
       - name: theme-provider
-        image: alfresco/alfresco-keycloak-theme:0.1
+        image: alfresco/alfresco-keycloak-theme:0.2-auth-363
```

Comment:
This image is being used for testing. It may need to be changed once AUTH-363 is complete.